### DISCUSSION TITLE
Allow sprites to be moved outside the Stage area

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#7350] [Ideas] [answered]
Scratch sprites are confined by the Stage borders. They cannot really move outside the Stage (so that young children don't loose them). So, to have sprites -that are moving towards the Stage borders- really move outside the Stage, you have to hide them. As this is not what the intuition says, it would be better if there was an addon that would allow Scratch sprites to really move to coordinates outside the Stage area so that they are fully not visible.

**Comment by BroJac5246** [comment]
Would this be while projects are running or when manually setting the coordinates from the sprite properties panel? The former wouldn't be compatible with vanilla Scratch, but the latter could be useful (especially if bundled with the ability to do so for size as well).

**Reply by sfederici** [reply]
My interest was for allowing this while the project is running. I understand that it would make the project not fully compatible with Scratch. If this can be done, the behaviour in Scratch would be the usual one, that is the sprite would be still partially visible on the border of the Stage.

**Reply by BroJac5246** [reply, marked as answer]
> My interest was for allowing this while the project is running. I understand that it would make the project not fully compatible with Scratch. If this can be done, the behaviour in Scratch would be the usual one, that is the sprite would be still partially visible on the border of the Stage.

I don't think this would be added because it would encourage people to create projects that don't work properly for most Scratch users.
